src/main/java/io/specto/hoverfly/junit/core/model/FieldMatcher.java

```diff
@@ -53,8 +59,15 @@ public String getXpathMatch() {
         return xpathMatch;
     }
 
+    public String getJsonPathMatch() {
+        return jsonPathMatch;
+    }
 
-    public static FieldMatcher fromString(String value) {
+    public String getXmlMatch() {
+        return xmlMatch;
+    }
+
+    static FieldMatcher fromString(String value) {
```

Comment:
This is not used for the DSL matcher, it is intended for upgrading v1 to v2 schema. That's why it is package private.